internal/topic/core_test.go

```diff
@@ -29,14 +29,28 @@ func TestCore_CreateTopic(t *testing.T) {
 	mockAdmin := messagebrokermock.NewMockBroker(ctrl)
 	topicCore := NewCore(mockTopicRepo, mockProjectCore, mockBrokerStore)
 	ctx := context.Background()
-	dTopic := getDummyTopicModel()
-	mockProjectCore.EXPECT().ExistsWithID(gomock.Any(), dTopic.ExtractedProjectID).Return(true, nil)
-	mockBrokerStore.EXPECT().GetAdmin(gomock.Any(), messagebroker.AdminClientOptions{}).Return(mockAdmin, nil)
-	mockAdmin.EXPECT().CreateTopic(gomock.Any(), messagebroker.CreateTopicRequest{dTopic.Name, DefaultNumPartitions}).Return(messagebroker.CreateTopicResponse{}, nil)
-	mockTopicRepo.EXPECT().Exists(gomock.Any(), common.GetBasePrefix()+"topics/test-project/test-topic")
-	mockTopicRepo.EXPECT().Save(gomock.Any(), dTopic)
-	err := topicCore.CreateTopic(ctx, dTopic)
-	assert.Nil(t, err)
+
+	tests := []struct {
+		topicModel *Model
+	}{
```

Comment:
Please fix formatting using a linter

Reply:
linter only made this change. This is how we have defined struct formatting in the past as well.
https://github.com/razorpay/metro/blob/master/internal/subscriber/subscriberimpl_test.go#L48